### PR TITLE
Change the default disk size from 120GB to 100GB

### PR DESCRIPTION
**Pull request by osherdp**
Since all product dependant places have been handled for the various changes in MGMT-9682, we can change the actual usage of it in our tests.

### CODE REVIEW

File: src/consts/resources.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 DEFAULT_MASTER_MEMORY: int = 16984
 DEFAULT_MTU: int = 1500
 DEFAULT_WORKER_DISK: int = 20 * GB
-DEFAULT_MASTER_DISK: int = 120 * GB
+DEFAULT_MASTER_DISK: int = 100 * GB
```
Comment (eliorerz) [inline src/consts/resources.py:7]:
```suggestion
DEFAULT_MASTER_DISK: int = DISK_SIZE_100GB
```

**Review by eliorerz — CHANGES_REQUESTED** [review]

File: src/consts/resources.py (hunk 1 above)
Comment (osherdp) [reply src/consts/resources.py]: Done!

**Review by eliorerz — APPROVED** [review]
LGTM

**Pull request merged**